src/libpsc/CMakeLists.txt

```diff
@@ -65,6 +65,10 @@ endif()
 if (PSC_HAVE_RMM)
   target_link_libraries(psc PUBLIC rapids::rmm)
 endif()
+if (PSC_HAVE_NVTX)
+  target_link_libraries(mrc PUBLIC CUDA::nvToolsExt)
+  set(HAVE_NVTX 1)
+endif()
```

Comment:
Why is this needed? nvtx should already be pulled in via libmrc.

Reply:
IDK previously when you first merged nvtx it wasn't included in libpsc and so I couldn't build until I added that.  I've been doing it ever since for new branches, maybe its not required now.

Reply:
I think it's not, so please remove it unless you find that it is needed after all.

Sorry for being a PITA about these bits ;)

Reply:
haha np, it is your code after all